Open Judge System/Web/OJS.Web/Controllers/TempController.cs

```diff
@@ -90,5 +96,12 @@ public ActionResult NormalizeParticipantsWithDuplicatedParticipantScores()
             result.Append("</ol>");
             return this.Content(result.ToString());
         }
+
+        // TODO: Remove this method after updating the entities
+        public void MigrateAllExistingLabsToNewEnumValue() =>
+            this.contestsData
+                .GetAllWithDeleted()
+                .Where(c => c.Type == (ContestType)2)
```

Comment:
IMO if c.Type is of type `ContestType` why not use == ContestType.OnlinePractialExam  instead of using casting ?

Reply:
Currently in the database there are no OnlinePractialExams in the sense of what the enumeration means and I think it is more understandable to use the number instead.